ios/ReactNativeVolumeController/ReactNativeVolumeController.m

```diff
@@ -40,6 +40,10 @@ - (void)stopObserving {
     hasListeners = NO;
 }
 
+- (dispatch_queue_t)methodQueue {
```

Comment:
this function is unused, and is just a wrapper around `dispatch_get_main_queue`, which you do already use directly.